### PR TITLE
chore(release): 2.30.6

### PR DESCRIPTION
**Pull request by CamdenFoucht**
### Features
* ASK State Updates ([#498](https://github.com/alexa/ask-cli/pull/408)) 

### Bug Fixes

* deploy skill package when icon uri path relative to its asset files ([#487](https://github.com/alexa/ask-cli/issues/487)) ([4b6ac2f](https://github.com/alexa/ask-cli/commit/4b6ac2ff8cd8b6511bdcf42938f05518ecf31361))
* display message property for all error instance types ([a0d2f19](https://github.com/alexa/ask-cli/commit/a0d2f19faeee36523be701522585272cff2a2d6d))
* display warning object message correctly ([#488](https://github.com/alexa/ask-cli/issues/488)) ([abbdc57](https://github.com/alexa/ask-cli/commit/abbdc574df25757990d2101d931cc6d063f64af2))
* use npm registry https url ([9e92ee5](https://github.com/alexa/ask-cli/commit/9e92ee5bf9aedcba1f683c1db877d0a9c1442cae))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.30.6](https://github.com/alexa/ask-cli/compare/v2.30.4...v2.30.6) (2023-12-09)
+
+
```
Comment (doiron) [inline CHANGELOG.md:10]: do you want to wait for the [new PR](https://github.com/alexa/ask-cli/pull/500) to be merged or want to follow up with another one to update the changelog?

**Review by doiron — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (CamdenFoucht) [reply CHANGELOG.md:10]: Lets merge your PR then update this one

Comment (doiron) [reply CHANGELOG.md:10]: extra stuff 

```
### Features

* add ASK_FORCE_ENABLE to allow skipping domain validation during enablement ([487f8db](https://github.com/alexa/ask-cli/commit/487f8dbcdc9429d04b6fd475aa4a2a8430831f1f))

```

Comment (doiron) [reply CHANGELOG.md:10]: updated the commit

**Pull request merged**